.travis.yml

```diff
@@ -0,0 +1,37 @@
+sudo: required
+language: java
+jdk: openjdk8
+services:
+  - docker
+python:
+  - "2.7"
+
+node_js: "node"
```

Comment:
Do you really need `node`?

Reply:
No :-) I'll remove that, just copied some ideas from Sarek ;-)  Why do you even have it there?

Reply:
It's for a bonus package thing that I came across which prints the amount of time taken between each line of stdoutput. So a way to see how long each pipeline step is taking.

It was a bit of a faff though and made the Travis output kind of hard to read (looks lovely locally).

https://github.com/paypal/gnomon

https://github.com/SciLifeLab/Sarek/pull/572/files

Reply:
Good catch, totally forgot that I had it there.
Thanks

The other problem with gnomon is that it was hiding the fact that there were problems in the test.

Reply:
> The other problem with gnomon is that it was hiding the fact that there were problems in the test.

This is fairly simple to fix with `/bin/bash -euo pipefail` (this is in all of our pipelines because piping to samtools etc had the same effect of changing the exit code).